node/silkworm/downloader/internals/preverified_hashes.hpp

```diff
@@ -41,25 +41,23 @@ namespace silkworm {
  * preverified_hashes_mainnet.cpp
  *
  */
+
 struct PreverifiedHashes {
     std::set<evmc::bytes32> hashes;  // Set of hashes of headers that are known to belong to canonical chain
-    uint64_t height{0};              // Block height corresponding to the highest preverified header
+    uint64_t height{0};              // Block height corresponding to the highest pre-verified header
 
     [[nodiscard]] bool contains(const evmc::bytes32& hash) const { return hashes.find(hash) != hashes.end(); }
 
     static PreverifiedHashes none;  // A void set of hashes that can be used to turn-off pre-verified hashes usage and
```

Comment:
Why define a static empty set of itself ?

Reply:
The WorkingChain must be able to run without pre-verified hashes. So by default it uses PreverifiedHashes::none.